web/book/src/reference/stdlib/compute/README.md

```diff
@@ -0,0 +1,19 @@
+# Compute
+
+Std operators can be used to declare new functions or plain variables
+
+## Examples
+
+```prql
+let is_adult = col -> col >= 18
+let is_cool = col -> (col | in ["PRQL", "Rust"])
+
+from employees
+select {
+    first_name,
+    last_name,
+    hobby,
+    adult = is_adult age,
+}
+filter (is_cool hobby)
+```
```

Comment:
Very good example! (Though maybe the irony of `is_cool` might be lost on some! Maybe we make that blander like "writes code" or something...)

I think we could put this at the top of https://prql-lang.org/book/reference/declarations/functions.html, rather than make a new page?